backends/edk2-compat/edk2-svc-generate.c

```diff
@@ -469,8 +391,7 @@ static int getOutputData(const unsigned char *buff, size_t size, struct Argument
 			rc = generateESL(buff, size, args, hashFunction, outBuff, outBuffSize);
 			break;
 		default:
-			prlog(PR_ERR, "ERROR: Unkown output format %s , see usage below...\n", args->outForm);
-			usage();
```

Comment:
You might be able to use `argp_help` here to print the usage message? Or have the new error message suggest trying `--help` or `--usage`...

Reply:
I do not think we can use any of the argp functions without supplying it with an ` struct argp_state *state` which we no longer have in these functions. However, I have added a `use  '--help' for more info`.

Reply:
oh right. No worries.